src/backend/function/Fusion.Resources.Functions/Functions/ProfileSync.cs

```diff
@@ -22,6 +24,24 @@ public ProfileSync(IHttpClientFactory httpClientFactory)
             resourcesClient = httpClientFactory.CreateClient(HttpClientNames.Application.Resources);
         }
 
+        [FunctionName("profile-sync-event")]
+        public async Task SyncProfile(
+            [EventSubscriptionTrigger(HttpClientNames.Application.People, "subscriptions/persons", "resources-profile")] MessageContext message,
+            ILogger log)
+        {
+            log.LogInformation("Profile sync event received");
+            log.LogInformation(message.Event.Data);
+            var body = message.GetBody<PeopleSubscriptionEvent>();
+
+            if (body.Type != PeopleSubscriptionEventType.ProfileUpdated) //manual filtering to avoid hard-coding in attribute
+                return;
+
+            var refreshResponse = await resourcesClient.PostAsJsonAsync($"resources/personnel/{body.Person.Mail}/refresh", new { });
```

Comment:
Should we maybe do a get first, to check if we even track the profile?

Reply:
The check is done in the refresh endpoint anyway. Given the amount of changes that "strangled" People i was thinking that it's better to reduce traffic to API?

Reply:
The ppl situation is a bit different, because that gets the whole dump from AD.. 

However the ppl service only sends out events for profiles that have actual changes we care about (not all ad properties that we care about, like licenses etc), and if the profile is tracked in the ppl database. This will not result in the same flooding.

Reply:
I guess one reason to maybe do a get first, is to reduce the amount of "legit" errors in AI logs? This would light up the 404 requests.. Dunno tbh.. But since the endpoint checks it should be good like this.

Reply:
Ok, leaving it like this for now.